common/scripted_triggers/wc_race_triggers.txt

```diff
@@ -72,9 +64,9 @@ can_have_sexual_relation_with_trigger = {
 							limit = {
 								OR = {
 									has_trait = creature_human
```

Comment:
Shouldn't everything use the `is_race_trigger`, instead of using has_trait? Just to make it consistant.

Reply:
Done in 577a4414aae73f5835b9a113eb734e9a26924832.